index.html

```diff
@@ -537,5 +537,71 @@ <h4 class="mb-0"><strong>Hacktoberfest 2022</strong></h4>
     </div>
   </div>
   <script src="./assets/javascript/app.js"></script>
+<div id="footer">
+  <div class="container"></div>
+<footer>
+  <!-- Footer main -->
```

Comment:
I don't think we need a footer like this with a lot of links. We just want a simple footer with text.